.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       with:
         submodules: recursive
     - name: Install dependencies
-      run: sudo apt-get update && sudo apt-get install libsdl2-dev libsdl2-mixer-dev libboost-dev libboost-program-options-dev g++-8
+      run: CUSTOM_SUDO="sudo" ./docker/ubuntu-deps.sh
```

Comment:
Would it work to run the script using something like `sudo bash ./docker/ubuntu-deps.sh`, and avoid the need for the `CUSTOM_SUDO` variable that way?

Reply:
Sure thing, unsure why I over-complicated it here. Was thinking about granularity. But it doesn't make much sense.

Fixed it now.